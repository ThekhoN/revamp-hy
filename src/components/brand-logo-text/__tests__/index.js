import React from 'react';
import renderer from 'react-test-renderer';
import BrandLogoText from '../index';

describe('BrandLogoText', () => {
  it('renders correct snapshot', () => {
    const tree = renderer.create(
      <BrandLogoText />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
