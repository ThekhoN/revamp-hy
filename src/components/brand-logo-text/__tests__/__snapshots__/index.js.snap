// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BrandLogoText renders correct snapshot 1`] = `
<div
  className="brand-logo__text"
>
  <span
    className="text--color-brand-dark-blue"
  >
    Home
  </span>
  <span
    className="text--color-brand-light-blue"
  >
    Yantra
  </span>
</div>
`;
